app/cp_nigeria/views.py

```diff
@@ -844,10 +844,7 @@ def cpn_constraints(request, proj_id, step_id=STEP_MAPPING["economic_params"]):
             form_errors = True
 
         if equity_form.is_valid():
-            equity_data = equity_form.save(commit=False)
-            equity_data.debt_start = scenario.start_date.year
-            equity_data.scenario = scenario
-            equity_data.save()
+            equity_data = equity_form.save()
```

Comment:
I deleted this code because now EquityData is already initialized on step 3 and the get_or_create method sets the debt_start and scenario as defined here, so it seemed duplicate

Reply:
I undid this because it did break things as I suspected it might when I made this comment :')